client/src/modules/Registry/components/AdditionalInfo/index.test.tsx

```diff
@@ -0,0 +1,95 @@
+/* eslint-disable testing-library/no-unnecessary-act */
+import { act, cleanup, fireEvent, render, screen } from '@testing-library/react';
+import { Form } from 'antd';
+import { ERROR_MESSAGES } from 'modules/Registry/constants';
+import { Course } from 'services/models';
+import { AdditionalInfo } from './index';
+
+const preferedCourses = [
+  {
+    id: 1,
+    name: 'test',
+    startDate: new Date().toUTCString(),
+    planned: true,
+  },
+] as Course[];
+
+const mockValues = {
+  preferedCourses,
+  languagesMentoring: ['English'],
+  dataProcessing: 1,
+  aboutMyself: "I'm Groot",
+};
+
+type Values = typeof mockValues;
+
+const previousHandler = jest.fn();
+const submitHandler = jest.fn();
+const submitFailedHandler = jest.fn();
+
+const setup = (values: Values) =>
```

Comment:
For readability please call function `renderAdditionInfo`

Reply:
fixed